src/main/java/com/amashchenko/maven/plugin/gitflow/AbstractGitFlowMojo.java

```diff
@@ -1057,7 +1065,7 @@ protected void mvnSetVersions(final String version) throws MojoFailureException,
                 executeMvnCommand(args.toArray(new String[0]));
 
                 String timestamp = getCurrentProjectOutputTimestamp();
-                if (timestamp != null && timestamp.length() > 1) {
+                if (timestamp != null && timestamp.length() > 1 && updateOutputTimestamp) {
```

Comment:
Can you move `updateOutputTimestamp` check before `getCurrentProjectOutputTimestamp()`

Reply:
Moved to enclosing `if`